ui/dashboards/src/views/ViewDashboard.tsx

```diff
@@ -49,8 +87,31 @@ export function ViewDashboard(props: ViewDashboardProps) {
               flexGrow: 1,
               overflowX: 'hidden',
               overflowY: 'auto',
+              display: 'flex',
+              flexDirection: 'column',
             }}
           >
+            {isEditMode ? (
+              <Button
+                sx={{
+                  alignSelf: 'flex-end',
+                }}
+              >
+                <AddIcon sx={{ fontSize: '1rem', marginRight: '8px' }} />
```

Comment:
A little worried with all the `fontSize: '1rem'` custom styles that I've seen for icons that maybe something's off with our theme or something? Just noticing a pattern.

Reply:
I removed all the custom styles for Icons and created overrides for it